### ISSUE TITLE
2.5.3 Broken -- AttributeError: Values instance has no attribute 'enabled_extensions'

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#474]
In GitLab by @asottile on Feb 11, 2016, 09:42

```
$ virtualenv venv -ppython2.7
Running virtualenv with interpreter /usr/bin/python2.7
New python executable in venv/bin/python2.7
Also creating executable in venv/bin/python
Installing setuptools, pip...done.
$ . venv/bin/activate
$ pip install flake8
Downloading/unpacking flake8
  Downloading flake8-2.5.3-py2.py3-none-any.whl
Downloading/unpacking pyflakes>=0.8.1,<1.1 (from flake8)
  Downloading pyflakes-1.0.0-py2.py3-none-any.whl (152kB): 152kB downloaded
Downloading/unpacking pep8>=1.5.7,!=1.6.0,!=1.6.1,!=1.6.2 (from flake8)
  Downloading pep8-1.7.0-py2.py3-none-any.whl (41kB): 41kB downloaded
Downloading/unpacking mccabe>=0.2.1,<0.5 (from flake8)
  Downloading mccabe-0.4.0-py2.py3-none-any.whl
Installing collected packages: flake8, pyflakes, pep8, mccabe
Successfully installed flake8 pyflakes pep8 mccabe
Cleaning up...
$ touch f.py
$ flake8 !$
flake8 f.py
Traceback (most recent call last):
  File "/tmp/venv/bin/flake8", line 11, in <module>
    sys.exit(main())
  File "/tmp/venv/local/lib/python2.7/site-packages/flake8/main.py", line 25, in main
    flake8_style = get_style_guide(parse_argv=True, config_file=DEFAULT_CONFIG)
  File "/tmp/venv/local/lib/python2.7/site-packages/flake8/engine.py", line 236, in get_style_guide
    _disable_extensions(kwargs['parser'], options)
  File "/tmp/venv/local/lib/python2.7/site-packages/flake8/engine.py", line 219, in _disable_extensions
    enabled = set(_parse_multi_options(options.enabled_extensions))
AttributeError: Values instance has no attribute 'enabled_extensions'
```

**Comment by asottile** [comment]
In GitLab by @bukzor on Feb 11, 2016, 09:54

mentioned in commit 7d40f525f0fde4c85b1ab76672a2f04b89c5b3ec

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Feb 11, 2016, 09:54

Fixed in 2.5.4
